src/services/expenseService.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable function-paren-newline */
+
+const expenses = [];
+
+const expenseServise = {};
+
+expenseServise.reset = () => {
+  expenses.length = 0;
+};
+
+expenseServise.getAll = (userId, categories, from, to) => {
+  let filteredExpenses = expenses;
+
+  if (userId) {
+    filteredExpenses = filteredExpenses.filter(
```

Comment:
DRY, use `filteredExpenses = filteredExpenses.filter(` only once, and move all `if`s inside it